docs/python/psycopg2/README.md

```diff
@@ -27,7 +27,7 @@
         - [With DB API Thread Safety](#with-db-api-thread-safety-1)
         - [With Driver Parameter Style](#with-driver-parameter-style-1)
         - [With libpq Version](#with-libpq-version-1)
-    - [With flask](#with-flask)
+        - [With Flask](#with-flask)
```

Comment:
This extra space would bring the flask link along the lines of `With libpq version`, which comes under `Results` section (https://open-telemetry.github.io/opentelemetry-sqlcommenter/python/psycopg2/#with-flask).

Reply:
@ymotongpoo Thanks for the PR!. Except this small indentation, things looks good to me.

Reply:
@sjs994 Oops, it was totally mistake. I misunderstood the section level. Fixed!